.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,38 +7,29 @@ are the most critical to review. -->
 
 ---
 
-### Author Checklist
+## Author Checklist
 
 *All items are required. Please add a note to the item if the item is not applicable and
 please add links to any relevant follow up issues.*
 
 I have...
 
 * [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
-* [ ] added `!` to the type prefix if API or client breaking change
 * [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#pr-targeting))
 * [ ] provided a link to the relevant issue or specification
-* [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/build/building-modules/01-intro.md)
+* [ ] reviewed "Files changed" and left comments if necessary
 * [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#testing)
 * [ ] added a changelog entry to `CHANGELOG.md`
-* [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
-* [ ] updated the relevant documentation or specification
-* [ ] reviewed "Files changed" and left comments if necessary
-* [ ] run `make lint` and `make test`
+* [ ] updated the relevant documentation or specification, including comments for [documenting Go code](https://blog.golang.org/godoc)
 * [ ] confirmed all CI checks have passed
 
-### Reviewers Checklist
+## Reviewers Checklist
 
 *All items are required. Please add a note if the item is not applicable and please add
 your handle next to the items reviewed if you only reviewed selected items.*
 
 I have...
 
 * [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
-* [ ] confirmed `!` in the type prefix if API or client breaking change
```

Comment:
why this one is getting removed?

Reply:
its not followed strictly, the changelog and upgrading md is sufficient here

Reply:
+1 for keeping it. It will be handy if we ever generate our changelog: https://github.com/cosmos/cosmos-sdk/blob/main/cliff.toml#L108-L109

Reply:
added back